docs/source/docker/examples/datasets_in_the_wild.rst

```diff
@@ -162,21 +162,20 @@ Let's have a look at some statistics of the two obtained datasets:
 We notice the following when looking at this table:
 
 - The **min distance** between two samples was 0 after ffmpeg selection whereas the
-  min distance significantly increased using coreset sampling.
+  min distance significantly increased using coreset selection strategy.
```

Comment:
put coreset in to ``

Reply:
See https://github.com/lightly-ai/lightly/pull/735#discussion_r847111684